samples/openapi3/client/petstore/python-experimental/docs/FakeApi.md

```diff
@@ -332,6 +333,64 @@ No authorization required
 
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
+# **get_additional_properties_with_array_of_enums**
+> additional_properties_with_array_of_enums.AdditionalPropertiesWithArrayOfEnums get_additional_properties_with_array_of_enums()
+
+Additional Properties with Array of Enums
+
+### Example
+
+```python
+from __future__ import print_function
+import time
+import petstore_api
+from petstore_api.api import fake_api
+from petstore_api.model import additional_properties_with_array_of_enums
+from pprint import pprint
+# Defining the host is optional and defaults to http://petstore.swagger.io:80/v2
+# See configuration.py for a list of all supported configuration parameters.
+configuration = petstore_api.Configuration(
+    host = "http://petstore.swagger.io:80/v2"
+)
+
+
+# Enter a context with an instance of the API client
+with petstore_api.ApiClient() as api_client:
+    # Create an instance of the API class
+    api_instance = fake_api.FakeApi(api_client)
+    
+    # example, this endpoint has no required or optional parameters
+    try:
+        # Additional Properties with Array of Enums
+        api_response = api_instance.get_additional_properties_with_array_of_enums()
```

Comment:
@jirikuncar why doesn't the documentation example pass in an instance of additional_properties_with_array_of_enums.AdditionalPropertiesWithArrayOfEnums?

Reply:
Never mind, the endpoint responds with that data type, it doesn't accept it